data.sql

```diff
@@ -91,3 +91,235 @@ UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name IN ('Angemon', 'Boarmon');
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Agumon', '02-03-2020', 0, true, 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Gabumon', '11-15-2018', 2, true, 8);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '02-08-2020', 0, false, -11), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '11-15-2021', 2, true, -5.7), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle', '04-02-1993', 3, false, -12.13), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', '06-12-2005', 1, true, -45), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Boarmon', '06-07-2005', 7, true, 20.4), 
+
```

Comment:
* Also some [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Insert%20the%20following%20data%20for%20specialties,Jack%20Harkness%20is%20specialized%20in%20Digimon.) are missing for specialization as it is required that you add these data below 👇 , Kindly add them

> Insert the following data for specialties:

> Vet William Tatcher is specialized in Pokemon.
> Vet Stephanie Mendez is specialized in Digimon and Pokemon.
> Vet Jack Harkness is specialized in Digimon.

Reply:
:green_circle: Great work on implementing these changes :+1: